fastavro/_schema_py.py

```diff
@@ -220,11 +220,13 @@ def parse_schema(
     if named_schemas is None:
         named_schemas = {}
 
+    if isinstance(schema, dict) and "__fastavro_parsed" in schema:
+        for key, value in schema["__named_schemas"].items():
+            named_schemas[key] = value
+
```

Comment:
@scottbelden As far as I can tell, the `_py.py` fallback implementation is currently not covered altogether resulting in changes (or additions) in them to always decrease code cov. Do you have any advice on how to handle this?

Reply:
It looks like the tests needed to be approved to run. I don't remember that being the case in the past but maybe github made some changes so that forks needs approval to run.

Reply:
Ah, alright. I was a bit confused, thanks for the explanation.